alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/field/endpoint/UploadFileButtonField.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * alert-common
+ *
+ * Copyright (c) 2019 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.common.descriptor.config.field.endpoint;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.synopsys.integration.alert.common.action.UploadEndpointManager;
+import com.synopsys.integration.alert.common.descriptor.config.field.ConfigField;
+import com.synopsys.integration.alert.common.descriptor.config.field.ConfigValidationFunction;
+import com.synopsys.integration.alert.common.enumeration.FieldType;
+
+public class UploadFileButtonField extends ConfigField {
+    private final String buttonLabel;
+    private final String endpoint;
+    private final List<ConfigField> subFields;
```

Comment:
After looking at the JS, it doesn't look like the subFields are actually being used. You can probably remove this field.